Solutions/AADUserInfo/Data Connectors/AADUserInfo/run.ps1

```diff
@@ -0,0 +1,96 @@
+# Input bindings are passed in via param block.
+param($Timer)
+
+# Get the current universal time in the default string format
+$currentUTCtime = (Get-Date).ToUniversalTime()
+
+# The 'IsPastDue' porperty is 'true' when the current function invocation is later than scheduled.
+if ($Timer.IsPastDue) {
+    Write-Host "PowerShell timer is running late!"
+}
+
+Import-Module AzureAD -UseWindowsPowerShell
```

Comment:
Getting error while testing. Please fix the issue.
![image](https://user-images.githubusercontent.com/78137909/121034050-577c0980-c7ca-11eb-9827-5a66719bfb46.png)

Reply:
How are you testing this? in Azure Functions?  I had to add -UseWindowsPowershell to get it to work in Azure Functions.  There is even a GH issue (here)[https://github.com/Azure/azure-functions-powershell-worker/issues/232] that says to do this in azure functions